packages/utils/scripts/buildRollup.ts

```diff
@@ -14,17 +14,3 @@ run('yarn rollup -c rollup.npm.config.js');
 // We want to distribute the README because it contains the MIT license blurb from Sucrase and Rollup
 fs.copyFileSync('src/buildPolyfills/README.md', 'build/cjs/buildPolyfills/README.md');
 fs.copyFileSync('src/buildPolyfills/README.md', 'build/esm/buildPolyfills/README.md');
-
```

Comment:
We can/should also get rid of the `transpile:uncached` command everywhere, that only existed for this reason!